README.md

```diff
@@ -1,70 +1,187 @@
+
+> Describe between 1-3 key features of the application.
+
+- **[Retrieve Stocks from the API]**
+- **[Display the list of countries in the home page]**
+- **[Filter the stocks by country]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://zippy-salamander-5d059d.netlify.app)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+run the following commands in your terminal
+
+- git clone https://github.com/erickhansx/react-capstone
+
+- git checkout main
+
+### Installation
+
+you can install the following linters into your proejct through node, by these commands:
+
+### - Linters installation:
+
+- Light House: npm install -g @lhci/cli@0.7.x
+- Webhint: npm install --save-dev hint@7.x
+- Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+- ESLint: npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+### - Installing Jest, Webpack, others ...
+
+- Node packages: npm install.
+
+### Testing
+
+you can test the web by the following commands:
+
+- npm run test.
+
+### Deployment
+
+you can deploy your project through github pages.
+
+- make a repo for your project on github.
+- go to your project settings.
+- click on pages.
+- select the desired branch to deploy.
+- click save.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Erick Hans Martinez**
+
+- GitHub: [@erickhansx](https://github.com/erickhansx)
+- Twitter: [@TheErickHans](https://twitter.com/TheErickHans)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project like and follow.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+

```

Comment:
- [ ] Your README is well documented 💯 However, I suggest giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👍

> Requirement ⬇
![image](https://user-images.githubusercontent.com/45231276/205420698-511b44c5-7519-447e-8ae3-298ab48ad6fb.png)

Reply:
Thank you for the observation, you are right. Is done! : )